example/android/build.gradle

```diff
@@ -5,7 +5,7 @@ buildscript {
         jcenter()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:2.2.3'
+        classpath 'com.android.tools.build:gradle:1.3.1'
```

Comment:
I agree with you, but I compared with tipsi-stripe project. Yes, if we decide use 2.2.3 I will change it. What do you think?